src/components/Form/__tests__/SearchInput.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import { randomId } from '../../../utils'
+
+import { Form } from '../'
+
+jest.mock('../../../utils')
+
+describe('<Form.SearchInput />', () => {
+  it('renders with a random id by default', () => {
+    randomId.mockImplementation(() => 'test-id')
+    const $ = shallow(<Form.SearchInput id={randomId()} />)
+    expect($.prop('id')).toBe('test-id')
+  })
+
+  it('takes additional props', () => {
+    const $ = shallow(<Form.SearchInput id="test-search" placeholder="Search..." value="foo" defaultValue="foo" />)
+    expect($.prop('id')).toBe('test-search')
+    expect($.prop('value')).toBe('foo')
+    expect($.prop('defaultValue')).toBe('foo')
+    expect($.prop('placeholder')).toBe('Search...')
+  })
+
+  it('takes an onChange prop', () => {
+    const testFn = jest.fn()
```

Comment:
This test doesn\'t really do anything.  You should test the it gets called when the component actually changes using enzyme simulate method.

Reply:
removed - will moved to root component test